src/projects/providers/webrtc/webrtc_provider.h

```diff
@@ -102,6 +102,10 @@ namespace pvd
 		std::shared_ptr<Certificate> CreateCertificate();
 		std::shared_ptr<Certificate> GetCertificate();
 
+		bool AddStream(const std::shared_ptr<WebRTCStream> &stream);
```

Comment:
AddStream / DeleteStream 

The function name conveys the misleading meaning of disposing of the Stream from the system. It is used to search the stream mapped to the session key in WHIP, so change it to another appropriate name.